cpp/src/io/comp/uncomp.cpp

```diff
@@ -498,6 +501,40 @@ size_t decompress_snappy(host_span<uint8_t const> src, host_span<uint8_t> dst)
   return uncompressed_size;
 }
 
+/**
+ * @brief ZSTD decompressor that uses nvcomp
+ */
+size_t decompress_zstd(host_span<uint8_t const> src, host_span<uint8_t> dst)
+{
+  auto stream = rmm::cuda_stream_default;
```

Comment:
Should we expose the `stream` parameter in API?

Reply:
yup! done.